circuit_knitting/utils/transforms.py

```diff
@@ -27,7 +27,7 @@
 from collections.abc import Sequence, Iterable, Hashable, MutableMapping
 from typing import NamedTuple
 
-from rustworkx import PyGraph, connected_components
+from rustworkx import PyGraph, connected_components  # type: ignore[attr-defined]
```

Comment:
I like this. I wonder how we can know when to remove it? Should we make an issue to keep up with this?

Reply:
ok sounds good!